internal/kafka/internal/services/kafka.go

```diff
@@ -1089,8 +1076,6 @@ func buildManagedKafkaCR(kafkaRequest *dbapi.KafkaRequest, kafkaConfig *config.K
 
 	if keycloakConfig.EnableAuthenticationOnKafka {
 		managedKafkaCR.Spec.OAuth = managedkafka.OAuthSpec{
-			ClientId:               kafkaRequest.SsoClientID,
-			ClientSecret:           kafkaRequest.SsoClientSecret,
```

Comment:
These https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/75fae9d3d049e58c06a7f70858de4fad03cf4a5c/openapi/kas-fleet-manager-private.yaml#L287-L290 needs to be removed as well. 
What do you think? @akoserwal @ziccardi

Reply:
yes